src/components/AvatarWithImagePicker.js

```diff
@@ -91,10 +86,7 @@ const propTypes = {
     /** File name of the avatar */
     originalFileName: PropTypes.string,
 
-    /** Whether navigation is focused */
-    isFocused: PropTypes.bool.isRequired,
```

Comment:
It should be the other way around
We need to keep the `isFocused` prop yype, but remove the `withNavigationFocusPropTypes` (it doesn't exist)